src/test/java/ReverseStringTest.java

```diff
@@ -9,5 +9,14 @@ void verifyTheTestForReversText(){
         Assert.assertEquals(ReverseString.reverseText( "Programming is not easy if you don't know how to swim..."), "...miws ot woh wonk t'nod uoy fi ysae ton si gnimmargorP");
     }
 
+    @Test
+    void verifyTestForReversingStringNull(){
+        Assert.assertNull(ReverseString.reverseText("Programming is not easy if you don't know how to swim..."), "");
+    }
 
+    @Test
+    void verifyTestForReversingStringNotNull(){
+        Assert.assertNotNull(ReverseString.reverseText("Programming is not easy if you don't know how to swim..."), "");
+
+    }
```

Comment:
This test is not testing anything useful.
Remove it please.

Try to always hand over "perfect work".
If this case was useful you would remove that empty line:
```suggestion
    void verifyTestForReversingStringNotNull(){
        Assert.assertNotNull(ReverseString.reverseText("Programming is not easy if you don't know how to swim..."), "");
    }
```